roundabout/userdefinedfields/models.py

```diff
@@ -49,7 +49,7 @@ def __str__(self):
 
 
 class FieldValue(models.Model):
-    field_value = models.TextField(null=True, blank=True)
+    field_value = models.TextField(null=False, blank=True)
```

Comment:
Let's revert this change to the Model field back to the original version. If we try to do a database migration like this, it will fail as the current fields that are Null will now violate this constraint. To do this, we need to first update existing data (either with another django migration or by shell), then change the schema.